packages/shared/src/components/post/widgets/CollectionsIntro.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import CloseButton from '../../CloseButton';
+import { ButtonSize } from '../../buttons/Button';
+import { useActions } from '../../../hooks';
+import { ActionType } from '../../../graphql/actions';
+
+export function CollectionsIntro(): ReactElement {
+  const { checkHasCompleted, completeAction } = useActions();
+
+  if (checkHasCompleted(ActionType.CollectionsIntro)) {
+    return null;
+  }
+
+  return (
+    <span className="flex relative flex-row p-3 pr-2 w-full rounded-10 border border-theme-color-cabbage">
+      <div className="flex flex-col flex-1 typo-callout">
```

Comment:
I think it's subhead, but unsure as the names seem mixed.
Asking [here](https://dailydotdev.slack.com/archives/C0381BQUEN9/p1701157996730849).

Reply:
@sshanzel Please note new comment in the thread